site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -40,6 +40,19 @@ When this value is not defined, the value defined by [field]#tenant.externalIden
 +
 This configuration is only utilized when [field]#application.multiFactorConfiguration.loginPolicy# is enabled.
 
+[field]#application.formConfiguration.adminRegistrationFormId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.20.0#::
+The unique Id of the form to use for Add and Edit User Registration in the FusionAuth admin UI.
++
+:premium_feature: custom forms
+include::../shared/_premium-edition-blurb-api.adoc[]
++
+:premium_feature!:
+
+[field]#application.formConfiguration.requireCurrentPasswordOnPasswordChange# [type]#[Boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.45.0#::
+When `true`, a user is required to present their current password when changing their password on the self-service account form.
++
+This setting has no effect when a self-service account form has not been set on the Application.
```

Comment:
We should also mention this is only enforced with the form has a password field configured.